product_listprice_list_view/__manifest__.py

```diff
@@ -1,9 +1,9 @@
 # Copyright 2019 chrono123 & Quartile Limited
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 {
     "name": "Product List Price Update List View",
-    "version": "12.0.1.0.1",
-    "license": "AGPL-3",
+    "version": "12.0.1.0.2",
+    "license": "LGPL-3",
```

Comment:
We should technically check with chrono123 on the change.

Reply:
No problem, I will let Frank knows about the change of the license